src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="cal-answer">
+      <div className="answer">0</div>
+    </div>
+    <button type="button" value="AC">
+      AC
+    </button>
+    <button type="button" value="+/-">
+      +/-
+    </button>
+    <button type="button" value="%">
```

Comment:
- [ ] According to requirements, `Calculator.js` should have at least 1 child component. There are some options available for you: For instance, you could create a reusable `<Button />` component and a `<DisplayAnswer />` component which makes your code even better 👍 😎